src/w3c/templates/headers.js

```diff
@@ -22,6 +22,20 @@ function getSpecTitleElem(conf) {
     specTitleElem.textContent = conf.title;
     specTitleElem.id = "title";
   }
+  if (conf.isPreview && conf.prNumber) {
+    if (conf.prUrl)
+      specTitleElem.prepend(
+        html`
+          <span
+            >Preview of PR <a href=${conf.prUrl}>#${conf.prNumber}</a>:
```

Comment:
```suggestion
            >Preview of PR <a href="${conf.prUrl}">#${conf.prNumber}</a>:
```

Reply:
As convention of the project, we like using the quotes for attributes (even if optional).

Reply:
My bad, how did this even work at all when I was testing? :fearful: